evergreen.yml

```diff
@@ -907,23 +907,45 @@ functions:
           # Confirm semgrep version
           semgrep --version
           set +e
-          timestamp=$(date +'%Y-%m-%dT%H_%M_%S')
-          semgrep --config p/rust --metrics=off --sarif --exclude "benchmark" --verbose --error --severity=ERROR --sarif-output=mongosqltranslate_$timestamp.sast.sarif > mongosqltranslate_$timestamp.sast.cmd.verbose.out 2>&1
+          semgrep --config p/rust --metrics=off --sarif --exclude "benchmark" --verbose --error --severity=ERROR --sarif-output=mongosql.sast.sarif > mongosql.sast.cmd.verbose.out 2>&1
```

Comment:
I change the name to be "mongosql.sast.sarif" because the scan is for all of the code base and I removed the timestamp in the name because it is problematic when we want to retrieve it later to publish as a release artifact.

Reply:
Should we keep the timestamp in the verbose file like we do in the schema manager?

From `schema-manager-rs`:
```
timestamp=$(date -u +'%Y-%m-%dT%H:%M:%SZ')
semgrep --config p/rust --metrics=off --sarif --exclude "bic-trip" --verbose --error --severity=ERROR --sarif-output=mongodb-schema-manager.sast.sarif > mongodb-schema-manager_$timestamp.sast.cmd.verbose.out 2>&1
```

Reply:
I just saw @bucaojit comment about the verbose file. Is this file only for the team's use?